docs/configuration/tenants.md

```diff
@@ -0,0 +1,41 @@
+All configuration options for a tenant are stored on the `tenant` record in the metadata database used by Supavisor.
+
+A `tenant` is looked via the `external_id` discovered in the incoming client connection.
+
+All `tenant` fields and their types are defined in the `Supavisor.Tenants.Tenant` module.
+
+## Field Descriptions
+
+`db_host` - the hostname of the server to connect to
+
+`db_port` - the port of the server to connect to
+
+`db_database` - the database of the Postgres instance
+
+`external_id` - an id used in an external system used to lookup a tenant
+
+`default_parameter_status` - the default initial connection parameters to use
+
+`ip_version` - the ip address type of the connection to the database server
+
+`upstream_ssl` - the ssl certificate to use when connecting to the database server
+
+`upstream_verify` - how to verify the ssl certificate
+
+`upstream_tls_ca` - the ca certificate to use when connecting to the database server
+
+`enforce_ssl` - enforce an SSL connection on client connections
+
+`require_user` - require at least one `user` is created for a tenant
```

Comment:
Required stored `users` with a pair of a username and password. As an opposite, only one user can be stored, through which the auth_query will be executed